doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,230 @@ temperature:
+
+- **Pressure Controller**: A pressure controller is designed to be used in conjunction with another 'master' flow controller, typically a MassFlowController. The master flow controller is installed on the inlet of the reactor, and the corresponding PressureController is installed on on outlet of the reactor. The PressureController mass flow rate is equal to the master mass flow rate, plus a small correction dependent on the pressure difference:
+
+ .. math:: \dot m = \dot m_{\rm master} + K_v(P_1 - P_2).
+
+
+Time Integration
+----------------
+
+Cantera provides an ODE solver for solving the stiff equations of reacting systems. If installed in combination with SUNDIALS, their optimized solver is used. Starting off the current state of the system, it can be advanced in time by two methods:
+
+- ``step()``: The step method computes the state of the system at the a priori unspecified time `t_{\rm new}`. The time `t_{\rm new}` is internally computed so that all states of the system only change within a (specifiable) band of absolute and relative tolerances. Additionally, the time step must not be larger than a predefined maximum time step `\Delta t_{\rm max}`. The new time `t_{\rm new}` is returned by this function.
+
+- ``advance``\ `(t_{\rm new})`: This method computes the state of the system at time `t_{\rm new}`, where `t_{\rm new}` describes the absolute time from the initial time of the system. By calling this method in a for loop for pre-defined times, the state of the system is obtained for exactly the times specified. Internally, several ``step()`` calls are typically performed to reach the accurate state at time `t_{\rm new}`.
+
+The use of the ``advance`` method in a loop has the advantage that it produces results corresponding to a predefined time series. These are associated with a predefined memory consumption and well comparable between simulation runs with different parameters. However, some detail (e.g. a fast ignition process) might not be resolved in the output data due to the typically large time steps.
+
+The ``step`` method results in much more data points because of the small timesteps needed. Additionally, the absolute time has to be kept tracked of manually.
+
+Even though Cantera comes pre-defined with typical parameters for tolerances and the maximum internal time step, the solution sometimes diverges. This is typically due to a too large value of the maximum time step. By reducing this value, convergence can often be achieved. However, this results in larger computation times. When computing reactor networks with variable time scales, the time step can also be changed on the fly (see also example `ic_engine.py <cython/examples/reactors_ic_engine.html>`_.).
```

Comment:
Is using too large a value for the maximum time step actually a problem? I'm not sure I've ever run into a situation where this happens.